packages/ui/src/composables/useSelectableList.ts

```diff
@@ -16,34 +17,56 @@ export const useSelectableListProps = {
 }
 
 export function useSelectableList (props: ExtractPropTypes<typeof useSelectableListProps>) {
-  const isStringOrNumber = (option: SelectableOption): option is (string | number) => {
-    const typeOfOption = typeof option
-    return typeOfOption === 'string' || typeOfOption === 'number'
+  const getOptionProperty = (option: SelectableOption, prop: StringOrFunction) => {
```

Comment:
This method can be outside of composable and this way we potentially can save a bit of memory.